test_util/aws.py

```diff
@@ -115,3 +125,116 @@ def get_public_agent_ips(self):
+                  'sa-east-1': 'ami-fd0f99e0',
+                  'us-east-1': 'ami-57cd8732',
+                  'us-west-1': 'ami-45844401',
+                  'us-west-2': 'ami-1255b321'},
+    'cent-os-7': {'ap-northeast-1': 'ami-965345f8',
+                  'ap-southeast-1': 'ami-332de750',
+                  'ap-southeast-2': 'ami-c80320ab',
+                  'eu-central-1': 'ami-1548ae7a',
+                  'eu-west-1': 'ami-2ea92f5d',
+                  'sa-east-1': 'ami-2921ad45',
+                  'us-east-1': 'ami-fa9b9390',
+                  'us-west-1': 'ami-12b3ce72',
+                  'us-west-2': 'ami-edf11b8d'},
+    'cent-os-7-dcos-prereqs': {'ap-northeast-1': 'ami-965345f8',
+                               'ap-southeast-1': 'ami-332de750',
+                               'ap-southeast-2': 'ami-c80320ab',
+                               'eu-central-1': 'ami-1548ae7a',
+                               'eu-west-1': 'ami-2ea92f5d',
+                               'sa-east-1': 'ami-2921ad45',
+                               'us-east-1': 'ami-fa9b9390',
+                               'us-west-1': 'ami-12b3ce72',
+                               'us-west-2': 'ami-edf11b8d'},
+    'coreos': {'ap-northeast-1': 'ami-84e0c7ea',
+               'ap-southeast-1': 'ami-84e0c7ea',
+               'ap-southeast-2': 'ami-f35b0590',
+               'eu-central-1': 'ami-fdd4c791',
+               'eu-west-1': 'ami-55d20b26',
+               'sa-east-1': 'ami-f35b0590',
+               'us-east-1': 'ami-37bdc15d',
+               'us-west-1': 'ami-27553a47',
+               'us-west-2': 'ami-00ebfc61'},
+    'debian-8': {'ap-northeast-1': 'ami-fe54f3fe',
+                 'ap-southeast-1': 'ami-60989c32',
+                 'ap-southeast-2': 'ami-07e3993d',
+                 'eu-central-1': 'ami-b092aaad',
+                 'eu-west-1': 'ami-0ed89d79',
+                 'sa-east-1': 'ami-a5bd3fb8',
+                 'us-east-1': 'ami-8b9a63e0',
+                 'us-west-1': 'ami-a5d621e1',
+                 'us-west-2': 'ami-3d56520d'},
+    'rhel-6': {'ap-northeast-1': 'ami-78379d78',
+               'ap-southeast-1': 'ami-faedeea8',
+               'ap-southeast-2': 'ami-7f0d4b45',
+               'eu-central-1': 'ami-8e96ac93',
+               'eu-west-1': 'ami-78d29c0f',
+               'sa-east-1': 'ami-d1d35ccc',
+               'us-east-1': 'ami-0d28fe66',
+               'us-west-1': 'ami-5b8a781f',
+               'us-west-2': 'ami-75f3f145'},
+    'rhel-7': {'ap-northeast-1': 'ami-35556534',
+               'ap-southeast-1': 'ami-941031c6',
+               'ap-southeast-2': 'ami-83e08db9',
+               'eu-central-1': 'ami-e25e6cff',
+               'eu-west-1': 'ami-8cff51fb',
+               'sa-east-1': 'ami-595ce844',
+               'us-east-1': 'ami-a8d369c0',
+               'us-west-1': 'ami-33cdd876',
+               'us-west-2': 'ami-99bef1a9'},
+    'ubuntu-12-04': {'ap-northeast-1': 'ami-88bca789',
```

Comment:
We won't ever support 12.04 or 14.04, so I don't see value in having them (16.04 will be supported eventually, so we could have that one).